kbase-extension/scss/partials/_dropzoneDz.scss

```diff
@@ -91,14 +92,23 @@
   .dz-message {
     color: rgb(0 0 0);
     font: normal 400 24px/28px $typeface-page-text;
-    margin: 2em 4.5em;
+    // margin: 2em 4.5em;
```

Comment:
Delete.

Reply:
Yeah, ye ole "not sure this will work, keep an eye on it and remove later" that doesn't get removed.
Gone, and also rearranging a bit to put styles where they should be, and adding some of those "this is the full path of this style" comments. They do help clarify, as nested styles can be painful to follow...

Reply:
Yeah, the nested styles are hard, having the full path as a comment is very helpful!